CONTRIBUTING.md

```diff
@@ -38,6 +39,15 @@ mamba env create -f dev-environment.yml  # Add '-n custom_name' if you want.
 mamba activate xdem-dev  # Or any other name specified above
 ```
 
+#### With `pip`
+```bash
+git clone https://github.com/GlacioHack/xdem.git
+cd xdem
+make install
+```
+
+Please note: pip installation is currently only possible under python3.10.
```

Comment:
Should we leave this README section to be only for users (for which `pip install xdem` should be sufficient?), and point towards the documentation for developer install?

Reply:
My bad, I just noticed this is already the dev installation in CONTRIBUTING!